docs/operator-manual/declarative-setup.md

```diff
@@ -1138,6 +1169,23 @@ stringData:
   tlsClientCertKey: ...
 ```
 
+Helm charts sourced from OCI registries should utilize the fields described previously as well as set the `enableOCI` field as `true`.
```

Comment:
So glad we are documenting this, I had an end user I support recently asking me where is the `enableOCI` documented.